src/ray/gcs/redis_context.h

```diff
@@ -73,7 +73,7 @@ class RedisContext {
                   int log_length = -1);
 
   Status SubscribeAsync(const ClientID &client_id, const TablePubsub pubsub_channel,
-                        const RedisCallback &redisCallback);
+                        const RedisCallback &redisCallback, int64_t *out_callback_index);
```

Comment:
Can you document this method and the arguments and say that the method writes the index of the callback to `out_callback_index`? This should have been done earlier by someone.